### PR TITLE
Add github issues template

### PR DESCRIPTION
**Pull request by prigoyal**
this morning while debugging user issues, I found myself asking these questions repeatedly, so creating a template to give out to people in future.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+For build issues, please add `[Build]` at the beginning of issue title.
+
+When submitting a bug report, please include the following information (where relevant):
+- OS:
+- How you installed TC (docker, conda, source):
+- Python version:
+- CUDA/cuDNN version:
+- Conda version (is using conda):
+- GCC/GXX version (if compiling from source):
+
```
Comment (wsmoses) [inline .github/ISSUE_TEMPLATE.md]: Add LLVM/Tapir version [or docker version]
Also add commit hash of our repo and submodules [if from source]

Ideally we make a script to generate all these versions and they copy output

Comment (prigoyal) [reply .github/ISSUE_TEMPLATE.md]: nice catch, updating

**Comment by prigoyal** [comment]
can I hazz accept on this one?  :)

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+When submitting a bug report, please include the following information (where relevant):
+- OS:
+- How you installed TC (docker, conda, source):
+- Python version:
+- CUDA/cuDNN version:
+- Conda version (is using conda):
+- Docker image (if using docker):
+- GCC/GXX version (if compiling from source):
+- Add LLVM/Tapir version:
```
Comment (ftynse) [inline .github/ISSUE_TEMPLATE.md]: you literally copied "_Add_ LLVM/Tapir version" :)

Comment (prigoyal) [reply .github/ISSUE_TEMPLATE.md]: hahahaha

**Comment by prigoyal** [comment]
me me @ftynse @wsmoses , review me :)

**Pull request merged**